tensorboard/context.py

```diff
@@ -32,9 +32,14 @@ class RequestContext(object):
 
     Fields:
       auth: An `AuthContext`, which may be empty but is never `None`.
+      remote_ip: An `ipaddress.IPv4Address` or `ipaddress.IPv6Address`, which may be None.
+        Best guess of the IP Address of the end user.
+      x_forwarded_for: A list of `ipaddress.IPv4Address` or `ipaddress.IPv6Address`,
+        which may be empty but is never None. This should be parsed value of x_forwarded_for
```

Comment:
nit: Parsed value of `X-Forwarded-For`, not `x_forwarded_for`.

Reply:
Done.